src/app/components/PostsList.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import Post from './Post';
+
+const PostsList = props => {
+  const [postsState, setPostsState] = useState([]);
+
+  useEffect(() => {
+    axios.get('http://jsonplaceholder.typicode.com/posts/')
+      .then(response => {
+        const result = response.data;
+        setPostsState(result)
+      }).catch(error => {
+        console.log(error)
+      });
+  })
```

Comment:
```suggestion
  }, [])
```
без этого мы будем бесконечно запрашивать у сервера посты, так как `setPostsState`, повторно вызывает перерисовку компонента, а вместе с этим и `setPostsState`